client/src/components/Footer/Footer.js

```diff
@@ -21,46 +23,68 @@ function Footer() {
       }
     }
   }, []);
+
+  const openChatWindow = () => {
+    const chatWindowFeatures =
+      'toolbar=no, location=no, directories=no, status=no, menubar=no, scrollbars=yes, resizable=yes, width=400, height=500, top=100, left=100';
+
+    const chatWindow = window.open('', 'Live Chat', chatWindowFeatures);
+
+    chatWindow.document.body.innerHTML = `
+      <iframe
+        title="Live Chat"
+        src="https://web.powerva.microsoft.com/environments/Default-99fe11df-ab57-448e-af64-07608c7218f7/bots/cr7c7_vanCityPropertyPulseBot/webchat?__version__=2"
+        style="width: 100%; height: 100%; border: none;"
+      ></iframe>
+    `;
+  };
+
   return (
-    isLogin && (
```

Comment:
we need `isLogin &&` to hide the footer when user is not logged in

Reply:
Fixed!